variables.tf

```diff
@@ -87,3 +87,9 @@ variable "access_log_bucket_name" {
   default     = ""
   description = "Name of the S3 bucket where s3 access log will be sent to"
 }
+
+variable "allow_ssl_requests_only" {
```

Comment:
```suggestion
variable "require_ssl" {
```

Reply:
Same comment as on the previous module...

in the underlying S3 storage module, there's a flag we're just passing through and it's called `allow_ssl_requests_only` in that module. We should keep the same name for consistency.

Reply:
Ok, agree